examples/ini.js

```diff
@@ -0,0 +1,47 @@
+const fs = require("fs");
+const ini = require("ini");
```

Comment:
Oh... I think `ini` will need to be added to `devDependencies`.

Reply:
All right, did that.